infra/core/database/azure-cache-for-redis.bicep

```diff
@@ -121,9 +141,23 @@ resource cache 'Microsoft.Cache/redis@2023-04-01' = {
       family: redisCacheFamily
       name: redisCacheSku
     }
+    redisConfiguration: {
+      'aad-enabled': 'true'
+    }
   }
 }
 
+@batchSize(1)
+resource redisCacheBuiltInAccessPolicyAssignment 'Microsoft.Cache/redis/accessPolicyAssignments@2023-08-01' = [for user in users: {
+  name: guid(resourceGroup().id, name, builtInAccessPolicyName, user.objectId)
+  parent: cache
+  properties: {
+    accessPolicyName: builtInAccessPolicyName
```

Comment:
This is where you would do the user.accessPolicyName to get it added to the right access policy.